cephfs/file.go

```diff
@@ -306,10 +305,10 @@ const (
 	// FallocNoFlag means default option.
 	FallocNoFlag = FallocFlags(0)
 	// FallocFlKeepSize specifies that the file size will not be changed.
-	FallocFlKeepSize = FallocFlags(C.FALLOC_FL_KEEP_SIZE)
+	FallocFlKeepSize = FallocFlags(0x01) // FALLOC_FL_KEEP_SIZE (include/linux/falloc.h)
```

Comment:
How about defining `FALLOC_FL_KEEP_SIZE` in the above `import "C"` with `#ifndef FALLOC_FL_KEEP_SIZE` guards?

Reply:
Yeah, I like that idea, will do that!

Reply:
i don't think defining these constant make sense. see https://github.com/ceph/go-ceph/pull/507#issuecomment-849514006 .

Reply:
But it is necessary to compile the code, or else we will have to disable the whole `Fallocate()` method in go-ceph on non-linux platforms, which I prefer not to do, because there is no technical reason for it.

Reply:
sorry, i missed your remarks while writing https://github.com/ceph/go-ceph/pull/507#issuecomment-849729740 . but in short, i agree with you.